components/ide/jetbrains/backend-plugin/build.gradle.kts

```diff
@@ -36,15 +36,30 @@ project(":") {
     kotlin {
         val excludedPackage = if (environmentName == "latest") "stable" else "latest"
         sourceSets["main"].kotlin.exclude("io/gitpod/jetbrains/remote/${excludedPackage}/**")
+
+        if (properties("platformType") == "RD") {
+            print("Rider: exclude unnecessary files")
+            sourceSets["main"].kotlin.exclude("**/GitpodForceUpdateMavenProjectsActivity.kt")
+            sourceSets["main"].kotlin.exclude("**/maven.xml")
+        }
     }
 
     sourceSets {
         main {
-            resources.srcDirs("src/main/resources-${environmentName}")
+            resources.srcDirs("src/main/resources")
+            if (properties("platformType") == "RD") {
+                print("Rider: import rider source set")
+                resources.srcDirs("src/main/resources-rider")
+            }
+            resources.srcDirs("src/main/resources-${environmentName.replace("-rider", "")}")
```

Comment:
How come this isn't within the if on line 50? I don't understand why we're doing a rider specific thing outside the IF block.

Reply:
Possible environmentNames:
- stable
- latest
- stable-rider
- latest-rider

I only need stable and latest to load resources-stable and resources-latest, do a simple replace is working well here